routes/route.py

```diff
@@ -15,12 +16,17 @@ def resource_not_found(e):
     return jsonify(error=str(e)), 400
 
 
-@routes.route('/test', methods=['GET'])
-def vernal():
-    print("Executing VERNAL similarity functions")
-    moduleLibraryPath = os.path.join(CURRENT_DIRECTORY, "../core/tools/GraphData/3dMotifAtlas_ALL_one_of_each_graph.cPickle")
-    tempResponsePath = os.path.join(CURRENT_DIRECTORY, "../core/tools/GraphData/response.json") #BP2 output
-    
-    return jsonify(graph_compare.k_most_similar_bp2(moduleLibraryPath, tempResponsePath))
+@routes.route('/CompareSequence/<dataset>', methods=['POST'])
```

Comment:
Just put the dataset as form data for consistency. There's no reason to have it as a query string.